incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
+    display: inline-block
```

Comment:
I don't think the mixin should set `display`. It feels unnecessary because this could be used in any number of ways and it would become a pain to always have to override the `display`.